src/main/java/uk/gov/justice/digital/hmpps/keyworker/config/JmsConfig.kt

```diff
@@ -94,4 +92,8 @@ open class JmsConfig {
     ))
     return awsSqsClient.getQueueUrl(queueName).queueUrl
   }
+
+  companion object: Logging {
+    val log = logger()
```

Comment:
why have you made the `log` public?  I can't see this is any better than the code it replaced - it is just less characters rather than adding value